Makefile

```diff
@@ -1,4 +1,5 @@
 cpu:
-	docker build . --file Dockerfile --no-cache --tag cr:cpu
+	docker build . --file Dockerfile --tag cr:cpu --build-arg BL_VERSION_SHORT="${BLENDER_VERSION_STRING}"
+
 gpu:
-	docker build . --file Dockerfile.cuda10 --no-cache --tag cr:gpu
+	docker build . --file Dockerfile.cuda10 --tag cr:gpu
```

Comment:
No worries mate ;) I just removed it and replaced this with a new docker file. Also, been a while hey! Took me some time to get around to finally doing this. Needed a reason and that reason may have just turned up.

Reply:
haha yes, I think overall this would need a bit more of a rework, because there is no need for CUDA specific containers anymore as long as ppl run an up to date version of docker and nvidia-docker2.

Also I think in order to minimize maintenance, one could generate Dockerfiles on demand, IIRC that was a long term goal anyway.